src/Plutarch/Extra/Value.hs

```diff
@@ -1,6 +1,12 @@
+{-# LANGUAGE DuplicateRecordFields #-}
+{-# LANGUAGE ExistentialQuantification #-}
+{-# LANGUAGE OverloadedRecordDot #-}
+{-# LANGUAGE PartialTypeSignatures #-}
+{-# LANGUAGE PolyKinds #-}
 {-# LANGUAGE TypeApplications #-}
 
 module Plutarch.Extra.Value (
+  -- TODO: Label these imports better
```

Comment:
I agree: no time like the present.

Reply:
Did a first pass here going by the comments and function names. May see some better opportunities for organization once I start looking at the actual functions themselves.